analysis_driver/pipelines/projects.py

```diff
@@ -1,9 +1,34 @@
-from egcg_core import rest_communication
-
+import glob
+import os
+from egcg_core import rest_communication, clarity
+from analysis_driver.config import default as cfg
+from analysis_driver.quality_control import Relatedness
+from analysis_driver.exceptions import PipelineError
 
 def project_pipeline(dataset):
+
     project_id = dataset.name
-    project = rest_communication.get_document('projects', where={'project_id': project_id})
-    # run the project pipeline
-    exit_status = 0
+    samples_for_project = rest_communication.get_documents('samples', where={'project_id': project_id})
+    species_in_project = []
+    for sample in samples_for_project:
+        sample_id = sample.get('sample_id')
+        species_in_lims = clarity.get_species_from_sample(sample_id)
+        species_in_project.append(species_in_lims)
+    if len(set(species_in_project)) != 1:
+        raise PipelineError('Wrong number of species in this project: expected 1')
+    species = list(set(species_in_project))[0]
+
+    working_dir = os.path.join(cfg['jobs_dir'], project_id)
+    reference = cfg['references'][species]['fasta']
+    project_source = os.path.join(cfg.query('sample', 'delivery_source'), project_id)
+    gvcf_paths = project_source + '/*/*g.vcf.gz'
```

Comment:
We take the risk of getting a gvcf that we do not want. We should search for the gvcf of each sample we have in ```samples_for_project```